lib/domain/libBlockchain/langs/yul/YulModel.h

```diff
@@ -189,6 +179,21 @@ namespace vanguard {
             }
         };
 
+        bool callNameMatches(const CallIns<Domain> &call , const std::string &regex) const {
```

Comment:
I think it makes more sense to declare `regex` as `const std::regex&`, since we only use `regex` to construct the `std::regex` instance anyways.
```suggestion
        bool callNameMatches(const CallIns<Domain> &call , const std::regex &reg) const {
```

Reply:
Fixed in https://github.com/Veridise/Vanguard/pull/57/commits/de5846dd6fc8fb87d79fecf7021a4cd3bce5c9dd